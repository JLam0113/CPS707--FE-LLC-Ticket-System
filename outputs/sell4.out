Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as admin, please enter a command
Sell tickets selected, please enter an event:
event1234567891012345678901234 selected, please enter the price:
Please enter the amount of tickets:
Invalid event title (too long), please enter a command.
Session ended
