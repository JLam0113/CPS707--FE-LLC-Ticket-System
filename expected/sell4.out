Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as admin, please enter a command
Sell tickets selected, please enter an event:
event1 selected, please enter the price:
Please enter the amount of tickets:
Invalid event title (too long), please enter a command.
Session ended
